leaderboard-ui/src/raiting.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "name": "klicunou",
+        "image": "https://scontent-frt3-2.xx.fbcdn.net/v/t1.0-1/c0.0.320.320a/p320x320/44827977_856027851454515_3991474135086661632_n.jpg?_nc_cat=107&_nc_ht=scontent-frt3-2.xx&oh=82af57d3bdaf47e98e04d5acd181eb95&oe=5D628F9C",
```

Comment:
Please explain what will happen if I change my facebook user picture.

Reply:
@KlicOgogo, yeah, this is temp solution, we should save and store photos here

Reply:
OK